### PR TITLE
Add Nexus/NXmpes documentation

### PR DESCRIPTION
**Pull request by domna**
This is just a proposal to start writing documentation. It could be as a standalone version to be integrated automatically into nomads mkdocs with the [mkdocs monorepo plugin](https://github.com/backstage/mkdocs-monorepo-plugin) or we can just eventually copy this over to nomad.

The benefit of maintaining this here is that we don't have to sync our PRs and can have everything in one place. It will then also get only updated in nomad when the submodule is updated, so we can also keep the appropriate version in the appropriate place (but some documentation may also not tied to a change in definitions/base classes). I also feel that github is a better place for external collaborators to join and review what we are doing.

TL;DR/I just want to see it: `pip install -r mkdocs-requirements.txt`, `mkdocs serve`, open http://localhost:8000 (yes, it live-reloads 😎)

@sanbrock @lukaspie @sherjeelshabih @mkuehbach @RubelMozumder @RonHildebrandt (btw is there a @FAIRmat/TeamB shortcut for this!? 😅)

### CODE REVIEW

**Comment by sherjeelshabih** [comment]
This looks really nice. Thanks for setting it up. I've made one quick change from all "nexus" in titles, etc, to "NeXus". I left the occurences in the notes as they are for now.
